estate/models/estate_property.py

```diff
@@ -0,0 +1,98 @@
+from odoo import fields, models, api, exceptions
```

Comment:
Order imports alphabetically and suggestion to avoid repetition.
```suggestion
from odoo import api, fields, models
from odoo.exceptions import UserError, ValidationError
```